torch/_functorch/config.py

```diff
@@ -178,6 +178,10 @@
 # of tensors in question.
 fake_tensor_propagate_real_tensors = False
 
+# This controls whether we collect donated buffer. This flag must be set
+# False if a user wants to retain_graph=True for backward.
+donated_buffer = False
```

Comment:
Hmm I was expected to see some logic in the `def backward()` in `runtime_wrappers.py` that checks if autograd is currently enabled in the backward, and if so raises an error if this config is True